src/olympia/lib/tests/test_remote_settings.py

```diff
@@ -16,114 +16,11 @@
 )
 class TestRemoteSettings(TestCase):
-    def test_setup_server_auth(self):
-        server = RemoteSettings('foo', 'baa')
-        responses.add(
-            responses.GET,
-            settings.REMOTE_SETTINGS_WRITER_URL,
-            content_type='application/json',
-            json={'user': {'id': ''}},
-        )
-        responses.add(
-            responses.PUT,
-            settings.REMOTE_SETTINGS_WRITER_URL + 'accounts/test_username',
-            content_type='application/json',
-            json={'data': {'password': 'test_password'}},
-            status=201,
-        )
-        server.setup_test_server_auth()
-
-        # If repeated then the account should exist the 2nd time
-        responses.add(
-            responses.GET,
-            settings.REMOTE_SETTINGS_WRITER_URL,
-            content_type='application/json',
-            json={'user': {'id': 'account:test_username'}},
-        )
-        server.setup_test_server_auth()
-
-    def test_setup_server_bucket(self):
-        server = RemoteSettings('foo', 'baa')
-        # if the server 403s on the bucket it's because it doesn't exist
-        responses.add(
-            responses.GET,
-            settings.REMOTE_SETTINGS_WRITER_URL + 'buckets/foo',
-            content_type='application/json',
-            status=403,
-        )
-        responses.add(
-            responses.PUT,
-            settings.REMOTE_SETTINGS_WRITER_URL + 'buckets/foo',
-            content_type='application/json',
-        )
-        # if the server 404s on the collection it's because it doesn't exist
-        responses.add(
-            responses.GET,
-            settings.REMOTE_SETTINGS_WRITER_URL + 'buckets/foo/collections/baa',
-            content_type='application/json',
-            status=404,
-        )
-        responses.add(
-            responses.PUT,
-            settings.REMOTE_SETTINGS_WRITER_URL + 'buckets/foo/collections/baa',
-            content_type='application/json',
-            status=201,
-        )
-        server.setup_test_server_collection()
-
-    def test_setup_server_collection(self):
-        server = RemoteSettings('foo', 'baa')
-        # But if the bucket exists then the collection should still be created
-        responses.add(
-            responses.GET,
-            settings.REMOTE_SETTINGS_WRITER_URL + 'buckets/foo',
-            content_type='application/json',
-        )
-        responses.add(
-            responses.GET,
-            settings.REMOTE_SETTINGS_WRITER_URL + 'buckets/foo/collections/baa',
-            content_type='application/json',
-            status=404,
-        )
-        responses.add(
-            responses.PUT,
-            settings.REMOTE_SETTINGS_WRITER_URL + 'buckets/foo/collections/baa',
-            content_type='application/json',
-            status=201,
-        )
-        server.setup_test_server_collection()
-
     @override_settings(REMOTE_SETTINGS_IS_TEST_SERVER=False)
```

Comment:
as this patch drops all support for `REMOTE_SETTINGS_IS_TEST_SERVER` actually doing anything, we should delete the setting in settings_base (and this override) too.

Reply:
https://github.com/mozilla/addons-server/pull/22724/commits/addf8ff91cd776ed2463373c5240343cf483cdae